src/Microsoft.Extensions.RuntimeEnvironment.Sources/project.json

```diff
@@ -0,0 +1,9 @@
+{
+  "version": "1.0.0-*",
+  "shared": "*.cs",
+  "dependencies": {},
+  "frameworks": {
+    "netstandard1.0": {},
+    "net451": {}
```

Comment:
I was just following the pattern we've been using for all the .Sources in this repo